lib/diagnostic.js

```diff
@@ -15,7 +15,14 @@ db.dropDatabase();
 // House Targaryen, motto 'Fire and Blood'
 
 // your code begins here
-
+db.houses.insert({
```

Comment:
The argument passed in here is an object with several duplicate keys.  Each document (represented as an object) should be inserted individually or as an array of objects.

```javascript
db.houses.insert({
  name: 'House Arryn',
  motto: 'As High as Honor'
});

db.houses.insert({
  name: 'House Stark',
  motto: 'Winter is Coming'
});

db.houses.insert({
  name: 'House Targaryen',
  motto: 'Fire and Blood'
});
```

or

```javascript
db.houses.insert([{
  name: 'House Arryn',
  motto: 'As High as Honor'
}, {
  name: 'House Stark',
  motto: 'Winter is Coming'
}, {
  name: 'House Targaryen',
  motto: 'Fire and Blood'
}]);
```